flow/src/main/java/com/sequenceiq/flow/service/flowlog/FlowLogDBService.java

```diff
@@ -264,17 +234,13 @@ public boolean repeatedFlowState(FlowLog lastFlowLog, String event) {
     }
 
     public void updateLastFlowLogPayload(FlowLog lastFlowLog, Payload payload, Map<Object, Object> variables) {
-        String payloadJson = JsonWriter.objectToJson(payload, writeOptions);
         String payloadJackson = JsonUtil.writeValueAsStringSilent(payload);
-        JsonUtil.checkReadability(payloadJackson, Payload.class);
-        String variablesJson = JsonWriter.objectToJson(variables, writeOptions);
         String variablesJackson = TypedJsonUtil.writeValueAsStringSilent(variables);
-        TypedJsonUtil.checkReadability(variablesJackson, Map.class);
         Optional.ofNullable(lastFlowLog)
                 .ifPresent(flowLog -> {
-                    flowLog.setPayload(payloadJson);
+                    flowLog.setPayload(null);
```

Comment:
Like above, why not set it to `"{}"`? (Irrelevant if the deprecated DB columns will be eventually dropped.)

Reply:
Got rid of these fields